addons/web/models/models.py

```diff
@@ -91,6 +91,15 @@ def web_save(self, vals, specification: dict[str, dict], next_id=None) -> list[d
         if next_id:
             self = self.browse(next_id)
         return self.with_context(bin_size=True).web_read(specification)
+    
+    def web_save_multi(self, vals: list[dict], specification: dict[str, dict]) -> list[dict]:
```

Comment:
Same than the create:
```suggestion
    def web_save_multi(self, vals_list: list[dict], specification: dict[str, dict]) -> list[dict]:
```